.github/workflows/checklinks.yml

```diff
@@ -0,0 +1,19 @@
+name: Check links
+
+on:
+  push:
+    branches:
+      - "**"
+
+jobs:
+  check-links:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Check links
+        uses: lycheeverse/lychee-action@v2
+        with:
+          args: --no-progress --verbose "**/README.md"
```

Comment:
Perhaps you can have this walk through all the different markdown files we have? Then we can check links throughout the book too?